jormungandr-lib/src/interfaces/vote.rs

```diff
@@ -405,7 +405,7 @@ impl TallyResult {
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub struct EncryptedTally(#[serde(with = "serde_base64_bytes")] Vec<u8>);
 
-mod serde_base64_bytes {
+pub mod serde_base64_bytes {
```

Comment:
I think we should start to move this kind of utilities to some other shared module ('serde_utils' or whatever). We can discuss it and I'll do it in a further PR.

Reply:
Yeah, but check also https://crates.io/crates/base64-serde

Reply:
I knew about that library. But, without having checked it properly maybe. I would say it doesn't support the check we do for the human-readable serializer in there.